go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/go/sampleLambda
+
+go 1.17
+
+replace github.com/thundra-io/thundra-lambda-agent-go/v2 => ./thundra-lambda-agent-go
```

Comment:
I think it's better if we keep this as a comment here. Or, maybe we should write a description of why it's needed. Because currently, `./thundra-lambda-agent-go` doesn't exist in this repository, so it might confuse someone.

Reply:
Maybe we should add a comment here?